### ISSUE TITLE
Background image loaded via insecure connection

### ISSUE THREAD

**Issue by fralix** [RestyaPlatform/board#2225]
## Expected Behavior
The background image of a board should be included via the same protocol (HTTPs vs HTTP) the other content is accessed. 

## Current Behavior
The custom board backgrounds are loaded via HTTP while other requests are made via HTTPs which leads to security warnings in most webbrowsers, i.e. Mozilla Firefox.

## Possible Solution
Make request URLs for backgrounds relative.

## Steps to Reproduce (for bugs)

1. Open Restyaboard and upload custom background image
2. Access the page via https
3. Check if the image was correctly loaded via https

## Context
There are security warnings in the browser (Parts of the side weren't loaded via the secure connection)

## Your Environment
Restya is running behind a **reverse proxy** accessing the software via local http connection!

Restya Version: 0.6.5
Reverse Proxy: lighttpd 1.5.51 (ssl)
Web Server: Apache2
Server OS: Debian Stretch, LVM container

**Comment by sridhar391** [comment]
@fralix Please change the link of the `Background image` of the image to `https` manually in the `boards table` of your `Restyaboard Database` and please change your Restyaboard link to `https`  in the `site_url_for_shell.php` file on `/usr/share/nginx/html/restyaboard/tmp/cache` path.

**Comment by rrjanbiah** [comment]
As @fralix mentioned, it is better to use relative path.. so that users don't have to manually update URLs like mentioned above

**Comment by fralix** [comment]
I agree. Most of my proxied apps are accessible via multiple (sub) domains, e.g. board.~ and restyaboard.apps.~ ... this is not necessary, but nice to have for my administration ... 

Other ressources are loaded correctly via https, so I assume it should be possible to extend this to **all** the linked ressources.

**Comment by rrjanbiah** [comment]
*Internal note*

This is not about Flickr for which #151 already fixed HTTPS issue

This is about uploading images in HTTP and accessing from HTTPS (after switching to HTTPS). So, have to use protocol relative URLs as in https://github.com/RestyaPlatform/board/blob/bd060dde606b35a8061e731ed67c9059f5a6fc32/client/js/views/modal_flickr_photo_view.js#L214
